articles/getting-started/requirements-for-sending-domains.md

```diff
@@ -13,29 +13,38 @@ The gold standard for a **Responsible Sender** is one who sends only to those re
 * Stops sending to recipients who have unsubscribed
 * Has a clear web presence, usually where a customer can sign up and manage their experience
 
-As part of being a **Responsible Sender,** we ask all customers to complete the sending domain validation process -- after which your domain(s) will be activated and ready for sending.  This process has two steps:
+As part of being a **Responsible Sender,** we ask all customers to complete the domain validation process -- after which your domain(s) will be activated and ready for use.  This process has two steps:
 
-1. Ownership Verification (DKIM or Abuse/Postmaster setup)
-1. Compliance Review
+### Step One: Ownership Verification
+
+First, we need to make sure you own the domain you want to set up. *Adding a DNS record* is the recommended way to show that you own your domain. Depending on how you plan to use the domain, you'll want to add different DNS records so you can verify ownership and get the domain properly set up all at once. [Learn why you should set up both a sending and a bounce domain]().
+
+* For a domain that you plan to use as a *sending domain* (From), you should add a TXT record with the provided DKIM information. This will make this domain "ready for sending and DKIM-signing", verifying ownership and at the same time resulting in better DKIM alignment.
+* For a domain that you plan to use as a *bounce domain* (Return Path), you should add a CNAME record with the provided hostname value. This will verify ownership and make this domain "ready for bounce".
+* If you don't have access to the domain's DNS records, you can verify ownership by requesting an email be sent to abuse@ or postmaster@ and then clicking the link in the received email. (Note: this email verification option is not available for bounce domains which _must_ be CNAME-verified.)
+
+### Step Two: Compliance Review
 
 ![Pending icon](media/requirements-for-sending-domains/pending-icon.png)
-Once ownership of your sending domain has been verified, the details are passed to our Compliance team who complete the review of your domain.  At this point your sending domain will have a status of "pending" for up to an hour until our compliance checks are complete. 
+Once ownership of your domain has been verified through any one of the above methods, the details are passed to our Compliance team who begin their review of your domain. At this point your sending domain will have a status of "Pending" for up to an hour until our compliance checks are complete. 
```

Comment:
Maybe instead of "details are passed to our Compliance team" something like "the domain enters our compliance process". Feels weird to specifically point out a human team.